php-checks/src/main/java/org/sonar/php/checks/formatting/FunctionSpacingCheck.java

```diff
@@ -44,14 +44,14 @@ private void checkClosureSpacing(FormattingStandardCheck formattingCheck, AstNod
     Token lParenToken = node.getFirstChild(PHPPunctuator.LPARENTHESIS).getToken();
     Token functionKeyword = node.getFirstChild(PHPKeyword.FUNCTION).getToken();
 
-    if (getNbSpaceBetween(functionKeyword, lParenToken) != 1) {
-      formattingCheck.reportIssue("Put exactly one space between the \"function\" keyword and the opening parenthesis.", node);
-    }
+//    if (getNbSpaceBetween(functionKeyword, lParenToken) != 1) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)